packages/trader/src/AppV2/Components/EntryExitDetails/entry-exit-details.scss

```diff
@@ -0,0 +1,27 @@
+.entry-exit-details {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-radius: 10px;
+
+    &__table {
+        display: flex;
+        justify-content: space-between;
+        flex-direction: column;
+
+        &-row {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+            border-bottom: 1px solid rgb(246, 247, 248);
+            margin: 8px 0;
```

Comment:
Could you please use quill token if it's possible in this case?

Reply:
done thanks